libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -685,7 +685,7 @@ export const DealTicket = ({
         subLabel={`${formatValue(
           normalizedOrder.size,
           market.positionDecimalPlaces
-        )} ${baseQuote} @ ${
+        )} ${baseQuote || ''} @ ${
```

Comment:
leaving this comment to prevent merging before QA review is done

Reply:
ah, and this line fixes this issue where `undefined` was displayed when no base quote tag was extracted
<img width="374" alt="Screenshot 2023-11-07 at 15 04 32" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/00551a73-0eb1-4ae5-a10c-7363b9a9ae5e">